### PR TITLE
set BUILD_PLATFORM and TARGET_PLATFORM on windows

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/core, this is ready for a review.

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/run_conda_forge_build_setup_win.bat [hunk 1]
```diff
@@ -103,6 +103,14 @@ if not "%CUDA_VERSION%" == "None" (
 )
 :: /CUDA
 
+conda.exe info --json | shyaml get-value platform > build_platform.txt
+set /p BUILD_PLATFORM=<build_platform.txt
+del build_platform.txt
+
+cat .ci_support\%CONFIG%.yaml | shyaml get-value target_platform.0 %BUILD_PLATFORM% > target_platform.txt
+set /p TARGET_PLATFORM=<target_platform.txt
+del target_platform.txt
```
Comment (hmaarrfk) [inline recipe/run_conda_forge_build_setup_win.bat:112]: I don't know batch scripting too well, but I couldn't find a better way to "store things in a batch variable".

Is it true that here you are this difficulty in storing things in a file, before reading in the file?

Comment (isuruf) [reply recipe/run_conda_forge_build_setup_win.bat:112]: Yes, there's another syntax to do this, but it's really unreadable. This technique is used in the same file just  few lines above.

**Pull request merged**